### PR TITLE
Fixed JavaScript heap out of memory

### LINKED ISSUE #234: Not possible to build with Microbundle
@blcham 

When building with Microbundle there is a `JavaScript heap out of memory` error. It also occurs in Github Actions (see https://github.com/kbss-cvut/s-forms/actions/runs/6719417410/job/18261003485).

Issue was reported here https://github.com/developit/microbundle/issues/870, and this solution seems to work https://github.com/developit/microbundle/issues/870#issuecomment-908563159, but we should track the resolution of this issue.

image

### PR DESCRIPTION
**Pull request by LaChope**
Resolves #234

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -109,7 +109,7 @@
   "scripts": {
     "test": "jest",
     "dev": "sb dev -p 6006",
-    "build": "microbundle --jsx React.createElement --jsxFragment React.Fragment",
+    "build": "export NODE_OPTIONS=--max_old_space_size=4096 && microbundle --jsx React.createElement --jsxFragment React.Fragment --target node",
```
Comment (blcham) [inline package.json:112]: Are u sure we need "--target node" ?

image

**Comment by blcham** [comment]
FYI based on comments from https://github.com/developit/microbundle/issues/870#issuecomment-908563159 I also investigated what is the size of the library (after `npm run build`) and it seems ok:
image

**Review by blcham — CHANGES_REQUESTED** [review]
See my comment about --target

File: package.json (hunk 1 above)
Comment (LaChope) [reply package.json:112]: Yes agree, I did not pay attention to it but you are right

**Comment by LaChope** [comment]
> FYI based on comments from [developit/microbundle#870 (comment)](https://github.com/developit/microbundle/issues/870#issuecomment-908563159) I also investigated the size of the library (after `npm run build`), and it seems reasonable (right?): image

Yes, seems ok to me

**Pull request merged**